ci/do_ci.sh

```diff
@@ -509,6 +509,11 @@ case $CI_TARGET in
         "${ENVOY_SRCDIR}/ci/upload_gcs_artifact.sh" /source/generated/docs docs
         ;;
 
+    docs-publish-latest)
+        BUILD_SHA=$(git rev-parse HEAD)
+        curl -X POST -d "$BUILD_SHA" "$NETLIFY_TRIGGER_URL"
+        ;;
```

Comment:
so where will the netlify to get the latest docs? github? gcs?

Reply:
from gcs - see https://github.com/envoyproxy/envoy-website/blob/63812b4a07d0b164059ebee2b267778e2ee7465c/docs/fetch_rst.py#L10-L20